skyvern/forge/agent.py

```
@@ -2352,6 +2352,33 @@ async def build_and_record_step_prompt(
+                        element_id=context.last_hovered_element_id,
+                        absolute_page_y=context.last_hovered_element_page_y,
+                        target_scroll_y=target_scroll_y,
+                    )
+                    await working_page.evaluate(f"window.scrollTo(0, {target_scroll_y})")
+                    # Small delay to let the scroll settle
+                    await asyncio.sleep(0.3)
+
+                    # Clear the saved position after scrolling back
+                    context.last_hovered_element_page_y = None
+                    context.last_hovered_element_id = None
+                except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)